src/pages/Bestellung.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+export default function Bestellung({ localStorage, inputData, totalPrice }) {
+  console.log(localStorage);
+  return (
+    <>
+      <Section>
+        <h4>Prüfe deine daten und bestätige den Kauf</h4>
+        <main>
+          <h4>Dein persönliches Data: </h4>
+          <h5>Name: {inputData.name}</h5>
+          <h5>Email: {inputData.email}</h5>
```

Comment:
Please use descending header elements. Never use the same in a row. That is not semantic. Read more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?retiredLocale=de) about it.